omod/package.json

```diff
@@ -3,8 +3,7 @@
   "version": "0.0.1",
   "description": "Mirebalais - Apppointment Scheduling Ui",
   "dependencies": {},
-  "devDependencies": {
-  },
```

Comment:
cc @dkayiwa  this seemed to be the only touched one but i resolved this thanks

Reply:
Many more: https://github.com/openmrs/openmrs-module-appointmentschedulingui/pull/18/files

Reply:
@dkayiwa  **the changes are too many for simply removing the extra Clear text.** Actually it wasnot about removing **clear** text however positioning a clear text  to the right position which i did. Formally as seen in the screenshot  on ticket clear text was just misplaced.

Reply:
Sure @dkayiwa  i explained a little bit  above thanks

Reply:
What is the use of the change in this file?

Reply:
So the idea about this text was only formatting and location in the change of both clear texts , the code was very ok but how it was formatted brought the clear text to be misplaced